src/pact.js

```diff
@@ -71,7 +71,7 @@ module.exports = (opts) => {
 
   logger.info(`Setting up Pact with Consumer "${consumer}" and Provider "${provider}" using mock service on Port: "${port}"`)
 
-  const mockService = new MockService(consumer, provider, port, host, ssl, pactfileWriteMode)
+  const mockService = new MockService(consumer, provider, dir, port, host, ssl, pactfileWriteMode)
```

Comment:
I think this is probably the cause of the failing build. You're passing it as the third argument, but in the MockService constructor it's the 6th.